From 5550af1fd10dccda80be4586f19e3aa0995be2bc Mon Sep 17 00:00:00 2001
From: Jan Blackquill <uhhadd@gmail.com>
Date: Wed, 17 Feb 2021 12:34:15 -0500
Subject: [PATCH] kcms/keyboard: fix migration

TriState::STATE_ON is 0, while TriState::STATE_OFF is 1.
We're currently migrating 0 -> disabled, when it should be
1 -> disabled, * -> enabled.

BUG: 431923
FIXED-IN: 5.21.1
---
 kcms/keyboard/kcminputrc_migrate_repeat_value.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/kcms/keyboard/kcminputrc_migrate_repeat_value.py b/kcms/keyboard/kcminputrc_migrate_repeat_value.py
index 1440c570e..bd1667969 100755
--- a/kcms/keyboard/kcminputrc_migrate_repeat_value.py
+++ b/kcms/keyboard/kcminputrc_migrate_repeat_value.py
@@ -4,7 +4,7 @@ import sys
 for line in sys.stdin:
     line = line.rstrip()
     print(line, file=sys.stderr)
-    if line.startswith("KeyboardRepeating=0"):
+    if line.startswith("KeyboardRepeating=1"):
         print("KeyRepeat=nothing")
     elif line.startswith("KeyboardRepeating="):
         print("KeyRepeat=repeat")
-- 
GitLab

